flocker/cli/script.py

```diff
@@ -250,9 +250,49 @@ def _changestate_on_nodes(self, deployment, deployment_config,
         return DeferredList(results)
 
 
+@flocker_standard_options
+class CLIOptions(Options):
+    """
+    Command line options for ``flocker`` CLI.
+
+    """
+    longdesc = """flocker does nothing (yet).
+
+    """
+
+    synopsis = ("Usage: flocker [OPTIONS] "
+                "\n"
+                "If you have any issues or feedback, you can talk to us: "
```

Comment:
:question: This feedback clause is repeated elsewhere and it is best that those remain consistent. Therefore I recommend using a variable.